harmony-kotlin/src/commonMain/kotlin/com.harmony.kotlin/data/repository/CacheRepository.kt

```diff
@@ -85,21 +86,24 @@ class CacheRepository<V>(
     return when (operation) {
       is DefaultOperation -> getAll(query, CacheSyncOperation())
       is MainOperation -> getMain.getAll(query)
-      is CacheOperation -> getCache.getAll(query).map {
-        try {
-          if (!validator.isValid(it)) {
+      is CacheOperation -> {
+        val cacheValues = getCache.getAll(query)
```

Comment:
getCache.getAll should be inside the try/catch. Same issue mentioned on get() method

Reply:
Fix applied to #60